newIDE/app/src/ObjectEditor/Editors/SpriteEditor/AnimationPreview.js

```diff
@@ -73,10 +73,7 @@ const AnimationPreview = ({
 }: Props) => {
   const forceUpdate = useForceUpdate();
 
-  // Use state for elements that don't need to be read from inside the animation callback.
-  const [fps, setFps] = React.useState<number>(
-    Math.round(1 / timeBetweenFrames)
-  );
+  const fps = 1 / timeBetweenFrames;
```

Comment:
Math.round()?

Reply:
I think it's a bit harmful: if you have an animation with >1s between frames, you end up with a FPS of 0 or 1